org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/tests/test.js

```diff
@@ -0,0 +1,65 @@
+/*jshint esversion: 6 */
+var opent2t = require('opent2t');
+var ava = require('ava');
+var config = require('./testconfig');
+var delay = require('delay')
+
+// This boilerplate code is copied from transpiled typescript AVA test code, which enables usage of 'yield' keyword. 
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {

```

Comment:
I'm concerned that this, and the use of __awaiter and yield below, makes the test code look far more complicated than it needs to be. We want other developers to be able to copy this test code to use for writing tests for translators they develop. If we're not using TypeScript, then we should just write the test code in normal ES6 syntax with .then() callbacks instead of awaits.